### DISCUSSION TITLE
Any examples of this being used inside Cloudflare Workers?

### DISCUSSION THREAD

**Discussion by balupton** [redom/redom#328] [Ideas]
Considering something simple for generating simple websites via cloudflare workers and their caching, rather than the behemoth of react.js complexity.

My concern with redom is about the mounting. A cloudflare worker supports the DOM API (so doesn't need `nodom`), but does not have a `window.document.body` to mount to. And there is no details inside the redom documentation on getting the `.outerHTML` of an element without mounting first.

**Comment by balupton** [comment]
Okay, editing the [Hello World JSFiddle](https://jsfiddle.net/mauroreisvieira/n3m5vajz/) works as expected:

``` javascript
const { el } = redom;
const hello = el("h1", "Hello world! ", el('h2', 'sup'));
console.log(hello.outerHTML);
```

So I'm assuming that the cloudflare worker can more or less do the same.

**Comment by pakastin** [comment]
If it has DOM support, should work. Otherwise just add some fake DOM. RE:DOM creates just ordinary DOM elements, no black magic 😎
